apps/products-feed/src/pages/api/feed/[url]/[channel]/google.xml.ts

```diff
@@ -219,6 +224,10 @@ export const handler = async (req: NextApiRequest, res: NextApiResponse) => {
     titleTemplate,
   });
 
+  logger.info("Generated XML size", {
+    size: xmlContent.length,
+  });
```

Comment:
we shouldnt generate logs just for that. You can attach "size" attribute to further logs